content/docs/tasks/security/vault-ca/index.md

```diff
@@ -154,3 +158,123 @@ between workloads using the certificates the Vault CA issued.
 After completing this tutorial, you may delete the testing cluster created
 at the beginning of this tutorial.
 
+## Configuring a basic Vault server
+
+The Vault server administrator is responsible of hosting and managing the server. When signing a CSR, Istio Citadel Agent
+passes the Kubernetes service account token of the workload and the CSR to the Vault server,
+which authenticates and authorizes the request (e.g., based on the
+[Vault Kubernetes `auth` method](https://www.vaultproject.io/docs/auth/kubernetes.html)) and
+returns the signed certificate to Istio Citadel Agent, if the request is authorized.
+Based on their security requirements, owners of Vault servers may configure various
+certificate issuance policies for Kubernetes service accounts and certificate
+signing requests.
+
+The following instructions configure an example basic Vault server to authenticate and authorize a CSR
+based on the Vault Kubernetes auth method.
+**The instructions here are for illustrative purposes only. Please consult with security experts
+to set up the security configuration and certificate issuance policies of your Vault servers.** The instructions are based on the
+posts [1](https://evalle.xyz/posts/integration-kubernetes-with-vault-auth/) and
+[2](https://coreos.com/tectonic/docs/latest/vault-operator/user/kubernetes-auth-backend.html).
+
+1.  Create a Kubernetes cluster to host an example basic Vault server.
+    In the Kubernetes cluster created, install, initialize, unseal, and login Vault.
+    Examples of install, initialize, unseal, and login Vault can be found in the post
+    [1](https://evalle.xyz/posts/integration-kubernetes-with-vault-auth/).
+    The example Vault server used in this guide is of 0.10.3 version.
```

Comment:
`0.10.3` i that the correct version? The latest version is 1.1.3.

Reply:
Yes, 0.10.3 is the version used in this example Vault server. The Vault API of version 0.10.3 is similar to that of the latest version.

Reply:
That should be clearer; this documentation mentions just "the example Vault server" without the term "API", whereas the Vault documentation generally refers to the product version. Specifically the [download page](https://www.vaultproject.io/downloads.html) references 1.1.3. Leaving only 0.10.3 in this documentation will confuse the user looking to download Vault version 0.10.3 which isn't available. I would suggesting stating the release version, and if you believe the API version  is important, *clearly* indicate that it's the API version.

Reply:
Revised to state the release version and added the link to the release.